cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -144,3 +144,19 @@ func (rc *ResultCounts) AddValidatingPolicyResponse(engineResponse engineapi.Eng
 		}
 	}
 }
+func (rc *ResultCounts) AddMutatingPolicyResponse(engineResponse engineapi.EngineResponse) {
```

Comment:
We also need a function similar to this to print the patched resource:
https://github.com/kyverno/kyverno/blob/eed287df300dfe1f0fd6da690acb426be3d5e5ae/cmd/cli/kubectl-kyverno/processor/policy_processor.go#L519-L527